src/Umbraco.Web.UI.Login/src/components/external-login-provider.element.ts

```diff
@@ -116,7 +116,7 @@ export class UmbExternalLoginProviderElement extends LitElement {
 					label="Continue with ${this.displayName}"
 					.look=${this.buttonLook}
 					.color=${this.buttonColor}>
-          ${this.displayName ? html`<div><uui-icon name=${this.icon}></uui-icon> Continue with ${this.displayName}</div>` : nothing}
+          ${this.displayName ? html`<div><uui-icon name=${this.icon} aria-hidden="true"></uui-icon> Continue with ${this.displayName}</div>` : nothing}
```

Comment:
@iOvergaard why doesn't `<uui-icon>` handle this?

E.g. like Shoelace icon component does:
https://github.com/shoelace-style/shoelace/blob/next/src/components/icon/icon.component.ts#L131-L139

Otherwise developers would need to remember to add this on their own.

Reply:
@bjarnef Although that is an excellent suggestion, that discussion is better targeted on the umbraco/umbraco.ui repository.